### PR TITLE
SAAS-987 add query resend

### PR DESCRIPTION
**Pull request by MaksimMalakhovSimbirsoft**
- added resend link;
- added service using RTK query, ~NEED FOR CORRECT ENDPOINT!!~;
SCREENFORSHARE

### CODE REVIEW

File: src/pages/ManageOrganization/MembersList/MembersList.service.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { createApi, fetchBaseQuery, BaseQueryFn, FetchArgs } from '@reduxjs/toolkit/query/react';
+import { ENDPOINTS } from 'core/api';
+import { oktaAuth } from 'core';
+import { MembersListStatusError, ResendEmailData } from './MembersList.types';
+
+const { Org } = ENDPOINTS;
+
+const baseQuery = fetchBaseQuery({
+  baseUrl: '',
```
Comment (TomasBlazek3PG) [inline src/pages/ManageOrganization/MembersList/MembersList.service.ts:9]: You are missing base url

File: src/pages/ManageOrganization/MembersList/MembersList.service.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+export const membersListApi = createApi({
+  reducerPath: 'membersListApi',
+  baseQuery,
+  refetchOnMountOrArgChange: true,
+  endpoints: (builder) => ({
+    createResendEmail: builder.mutation<any, ResendEmailData>({
+      query: ({ organizationId, memberId }: ResendEmailData) => ({
+        method: 'POST',
+        url: Org.resendEmail(organizationId, memberId),
+        // body: { organizationId, memberId },
```
Comment (TomasBlazek3PG) [inline src/pages/ManageOrganization/MembersList/MembersList.service.ts:34]: Why is it here?

File: src/pages/ManageOrganization/MembersList/MembersList.tsx [hunk 3]
```diff
@@ -34,7 +45,15 @@ export const MembersList: FC = () => {
                 </div>
               ) : (
                 <div className={styles.clockIconWrapper} data-testid="user-not-activated">
-                  <Clock className={styles.clockIcon} />
+                  <div>
+                    <Clock className={styles.clockIcon} />
+                    <span className={styles.labelsResendEmails}>{Messages.pendingConfirmation}</span>
+                  </div>
+                  <p>
+                    <a className={styles.link} onClick={() => handleResend(orgId, memberId)}>
```
Comment (TomasBlazek3PG) [inline src/pages/ManageOrganization/MembersList/MembersList.tsx:53]: For this we are using `<LinkButton variant="link" />`

**Review by TomasBlazek3PG — CHANGES_REQUESTED** [review]

File: src/pages/ManageOrganization/MembersList/MembersList.service.ts (hunk 1 above)
Comment (nicolalamacchia) [reply src/pages/ManageOrganization/MembersList/MembersList.service.ts:9]: that's correct, we don't need a base URL in this case, we can leave it as it is for now

**Review by nicolalamacchia — DISMISSED** [review]

File: src/pages/ManageOrganization/MembersList/MembersList.tsx (hunk 3 above)
Comment (nicolalamacchia) [reply src/pages/ManageOrganization/MembersList/MembersList.tsx:53]: that's correct, we don't need a base URL in this case, we can leave it as it is for now

File: src/pages/ManageOrganization/MembersList/MembersList.service.ts (hunk 1 above)
Comment (nicolalamacchia) [reply src/pages/ManageOrganization/MembersList/MembersList.service.ts:9]: that's correct, we don't need a base URL in this case, we can leave it as it is for now

File: src/pages/ManageOrganization/MembersList/MembersList.service.ts (hunk 2 above)
Comment (MaksimMalakhovSimbirsoft) [reply src/pages/ManageOrganization/MembersList/MembersList.service.ts]: I uncommented this

**Comment by MaksimMalakhovSimbirsoft** [comment]
Снимок экрана от 2022-08-09 20-00-13
added conditions to resending... You must be admin role and member status must be PROVISIONED. Well maybe for testing I could remove member status condition and this allows to check resend link

**Review by TomasBlazek3PG — APPROVED** [review]

File: src/core/api/endpoints.ts [hunk 4]
```diff
@@ -18,6 +18,8 @@ export const ENDPOINTS = {
     editMember: (orgId: string, memberId: string) => `/v1/orgs/${orgId}/members/${memberId}`,
     removeMember: (orgId: string, memberId: string) => `/v1/orgs/${orgId}/members/${memberId}`,
     inviteMember: (orgId: string) => `/v1/orgs/${orgId}/members`,
+    // Resend Email (Required correct endpoint)
+    resendEmail: (orgId: string, memberId: string) => `/v1/orgs/${orgId}/members/${memberId}`,
```
Comment (nicolalamacchia) [inline src/core/api/endpoints.ts:22]: Is this the correct endpoint? if so, remove the comment

Comment (MaksimMalakhovSimbirsoft) [reply src/core/api/endpoints.ts:22]: Oh, sure. It is correct.

Comment (MaksimMalakhovSimbirsoft) [reply src/core/api/endpoints.ts]: Done

**Review by TomasBlazek3PG — APPROVED** [review]

**Review by nicolalamacchia — DISMISSED** [review]

File: src/pages/ManageOrganization/MembersList/MembersList.tsx [hunk 5]
```diff
@@ -42,7 +42,7 @@ export const MembersList: FC = () => {
                     <Clock className={styles.clockIcon} />
                     <span className={styles.labelsResendEmails}>{Messages.pendingConfirmation}</span>
                   </div>
-                  {orgRole === 'Admin' && status !== MemberStatus.provisioned && (
+                  {orgRole === 'Admin' && status === MemberStatus.provisioned && (
```
Comment (TomasBlazek3PG) [inline src/pages/ManageOrganization/MembersList/MembersList.tsx:45]: Please use the admin string defined in ROLES in the constants

**Review by TomasBlazek3PG — CHANGES_REQUESTED** [review]

File: src/pages/ManageOrganization/MembersList/MembersList.tsx (hunk 5 above)
Comment (MaksimMalakhovSimbirsoft) [reply src/pages/ManageOrganization/MembersList/MembersList.tsx:45]: ok

**Review by TomasBlazek3PG — APPROVED** [review]

**Review by TomasBlazek3PG — APPROVED** [review]

**Review by nicolalamacchia — APPROVED** [review]

**Pull request merged**